xmtp_mls/src/groups/mls_ext/commit_log_storer.rs

```diff
@@ -192,21 +189,22 @@ impl CommitLogStorer for MlsGroup {
     }
 
     fn mark_failed_commit_logged<Db: ConnectionExt>(
+        &self,
```

Comment:
Unrelated simplification that was unlocked by a previous refactor, it was a tiny change so I just included it here.